### ISSUE TITLE
Reproduction gap between the released checkpoint and Table 1 on the full test split

### ISSUE THREAD

**Issue by pepper725** [xg-chu/UniLS#3]
Hi, thank you for releasing the code, checkpoint, and dataset.

I evaluated the released `UniLSLoRAGen.pt` checkpoint using the official `eval.py` on the complete Seamless Interaction test split. The result is generally close to Table 1, but several metrics—especially `L_FDD` and `L_JDD`—show a noticeable gap.

## Reproduction setup

- Checkpoint: `UniLSLoRAGen.pt`

- Test split: all 3,457 valid test records
- GPU: NVIDIA A100-SXM4 40GB
- Seed: `42`, as set by the official `eval.py`
- Inference parameters:
  - `tau=1.0`
  - `cfg=0.3`
- Batch size: `1`
- No smoothing, custom batching, metric modifications, or other post-processing was applied.
- The external dataset YAML only changes `DATA_PATH` and `META_PATH` to the local downloaded LMDB and metadata. Other fields are identical to the released `configs/dataset.yaml`.

Command:

```bash
CUDA_VISIBLE_DEVICES=0 python eval.py \
  --resume_path /path/to/UniLSLoRAGen.pt \
  --dataset ./configs/dataset_local.yaml \
  --tau 1.0 \
  --cfg 0.3
```

The official script evaluated all records successfully:

```text
100% 3457/3457
```

## Results

| Metric ↓ | Table 1 | Released code/checkpoint |
|---|---:|---:|
| S_LVE | 5.83 | 6.1806 |
| S_MHD | 1.89 | 1.9782 |
| S_FDD | 18.41 | 19.9053 |
| S_PDD | 4.67 | 4.6780 |
| S_JDD | 0.71 | 0.7123 |
| L_FDD | 17.12 | 18.8662 |
| L_PDD | 4.75 | 4.4764 |
| L_JDD | 0.98 | 1.1190 |

Raw output:

```text
[tau=1.0, cfg=0.3]
S_LVE=6.1806
S_MHD=1.9782
S_FDD=19.9053
S_PDD=4.6780
S_JDD=0.7123
L_FDD=18.8662
L_PDD=4.4764
L_JDD=1.1190
```

Some metrics, such as `S_PDD` and `S_JDD`, reproduce Table 1 very closely, and `L_PDD` is even better. However, the differences are larger for `S_LVE`, `S_FDD`, `L_FDD`, and `L_JDD`.

## Questions

Could you please clarify the following?

1. Is the released `UniLSLoRAGen.pt` exactly the checkpoint used for Table 1?
2. Were the Table 1 numbers obtained using a single seed, an average over multiple seeds, or the best result among several seeds?
3. What exact random seed was used for Table 1?
4. Are `tau=1.0` and `cfg=0.3` the exact parameters used for Table 1?
5. In `UniLSGen.inference`, classifier-free guidance appears to be applied only when `cfg > 1.0`:

```python
if cfg > 1.0:
    motion_logits = (
        cfg * motion_logits[:batch_size]
        + (1 - cfg) * motion_logits[batch_size:]
    )
else:
    motion_logits = motion_logits[:batch_size]
```

Does this mean that `cfg=0.3` intentionally disables CFG and uses only the conditional branch, or was a different value/implementation used for the paper?

6. Was any post-processing, temporal smoothing, sample selection, or alternative style reference used when producing Table 1?
7. The released evaluator reports eight metrics but not `F-FID` or `P-FID`. Could you also release or describe the official FID evaluation protocol?
8. During model loading, the official script prints:

```text
Missing keys: ['audio_encoder']
```

Is this expected for the released checkpoint?

If possible, could you provide the exact evaluation command, checkpoint hash, seed, and environment used to produce Table 1?

Thank you for your help and for releasing the project.
